src/pages/workspace/companyCards/WorkspaceCompanyCardsPage.tsx

```diff
@@ -35,8 +35,8 @@ function WorkspaceCompanyCardPage({route}: WorkspaceCompanyCardPageProps) {
     const selectedFeed = CardUtils.getSelectedFeed(lastSelectedFeed, cardFeeds);
     const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${selectedFeed}`);
 
-    const isLoading = !cardFeeds || !!(cardFeeds.isLoading && !cardFeeds.settings);
     const companyCards = CardUtils.removeExpensifyCardFromCompanyCards(cardFeeds?.settings?.companyCards);
+    const isLoading = !cardFeeds || !!(cardFeeds.isLoading && !companyCards);
```

Comment:
@waterim @getusha It looks like the `companyCards` will never be a falsy value, so I'm not sure if this check is correct. Could you double-check?

Reply:
We want only initial loading for a smooth work

Reply:
But it looks like `!companyCards` will never be true

Reply:
I think you are right, lets try to fix this in a next PR